### ISSUE TITLE
Image resize on frontend upload

### ISSUE THREAD

**Issue by danjde** [geqo/JoomlaImageResize#1]
Hi geqo,
I'm searching a plugin to permit to resize/crop and optimize images frontend uploaded on Joomla.
Especially I'm using Event Booking and would like enable  automatic frontend upload resized/cropped/optimized submission on frontend user submission.
Is this possible with your plugin?

Many thanks!

Davide

**Comment by geqo** [comment]
I do not have Event Booking, so I do not know how to extend my plugin for use in this extension. So far, it only supports default com_media

**Comment by danjde** [comment]
Perfect, thanks!

Davide

**Comment by geqo** [comment]
> Perfect, thanks!
> 
> Davide

But if you can give me it, i can try to upgrade plugin.

**Comment by danjde** [comment]
Hi @geqo,
yest I think it could be possible, the software should be OpenSource, only the support for a fee.
I've not the latest version but last year's version, and I know that the upload interface use the Joomla native system.
Anyway, what I'm looking for is a system that resize, crop and optimize images just after loading (jpg normally) overwriting the ones loaded by the user.
This is because then there are modules that take the same images and other components that use it.
For this reason it would be useful that the images should be corrected in the desired dimensions just after loading (into media directory). 
I have not understood well if your plugin works in this sense :-)

Thanks!

Davide

**Comment by geqo** [comment]
> I have not understood well if your plugin works in this sense :-)

Plugin resizes the image proportionally uses only **width** setting. When an image is uploaded, plugin creates a smaller copy of the image in the directory /images/SIZE/image.jpg|png|etc, where the SIZE - thumnail setting. In this case original image isn't deleted, but stored in the directory where it is loaded by default.

**Comment by geqo** [comment]
>Hi @geqo,
yest I think it could be possible, the software should be OpenSource, only the support for a fee  


It costs 40$

**Comment by danjde** [comment]
What I need, should be an overwrite file option, maintaining same path and same file name.
I don't understand your your previous post. Do you want to send you the files or not? 

Thanks

Davide

**Comment by geqo** [comment]
Yep, i need to see the file download code to see if I can add support for this extension.
